pkg/webui/console/views/application-api-key-add/index.js

```diff
@@ -66,6 +67,13 @@ export default class ApplicationApiKeyAdd extends React.Component {
     this.createApplicationKey = key => api.application.apiKeys.create(props.appId, key)
   }
 
+  static propTypes = {
+    appId: PropTypes.string.isRequired,
+    navigateToList: PropTypes.func.isRequired,
+    pseudoRights: PropTypes.rights.isRequired,
+    rights: PropTypes.rights.isRequired,
+  }
+
```

Comment:
I'd prefer to have all static props always at the very top of the class definition (so even before the constructor).